media_driver/agnostic/gen11/codec/kernel/media_srcs.cmake

```diff
@@ -18,9 +18,19 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 
-set(TMP_SOURCES_
-    ${CMAKE_CURRENT_LIST_DIR}/igcodeckrn_g11.c
-)
+if(${FULL_OPEN_BUILD} STREQUAL "yes")
+
+    set(TMP_SOURCES_
+        ${CMAKE_CURRENT_LIST_DIR}/igcodecdummykrn_g11.c
```

Comment:
Ok, why we define so many identical _dummy_ files for each platform?! This does not make sense to me and complicates everything. For example, it is hard to review which kernels are dummy and which are not. Could we PLEASE have single file for the dummy kernels? And we can use `#define IGCODECKRN_G11_SIZE IGCODECKRN_G10_SIZE` in it or some c++ alias trick.

Code should be simple. Right now driver has so many duplication...  

IMHO, just _look_ in the size of this _trivial_ PR! It is huge while actually it can be made visible on just 2 screens.